pages/03.installation/01.environment/01.homestead/docs.md

```diff
@@ -79,6 +79,11 @@ cd userfrosting
 git clone -b v7.20.0 https://github.com/laravel/homestead.git vagrant/Homestead
 ```
 
+You will need a SSH key-pair to connect to the Virtual Machine created in the next step. 
+```sh
+ssh-keygen -t rsa -f $HOME/.ssh/homestead_rsa
```

Comment:
Actually, this should probably be `.ssh/id_rsa`, the same as here : https://github.com/userfrosting/UserFrosting/blob/master/vagrant/bootstrap.yaml#L12

Also, does this will work on Windows too?

Reply:
Hmm, I thought I had already changed that but will update it now.

Should it also include any instructions regarding a password not being necessary, such as here: https://learn.userfrosting.com/installation/other-situations/homestead#create-an-ssh-keypair

This does work on Windows.

Reply:
The passphrase? I guess. For default key, some people may want to create one, but if someone need help creating an ssh key, it probably won't matter anyway. Or we add the ssh key thing in a 'notice' block or whatever ?